php/send_message/index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+require 'vendor/autoload.php';
+
+include_once 'src/DiscordChannel.php';
+include_once 'src/SMSChannel.php';
+include_once 'src/EmailChannel.php';
+include_once 'src/TwitterChannel.php';
+include_once 'src/NullChannel.php';
+
+return function($req, $res) {
+    try {
+        $payload = \json_decode($req['payload'], true);
+    } catch (\Exception $e) {
+        $res->json([
+            'status' => false,
+            'message' => $e->getMessage(),
+        ]);
+    };
+
+    $notificator = match (strtolower($payload['type'])) {
```

Comment:
There's a warning if an array key that doesn't exist is accessed:

```
Warning: Undefined array key \"type\" in /usr/code-start/index.php on line 21
```

Would you please improve this to not have warnings?

Reply:
Resolved in https://github.com/open-runtimes/examples/pull/62/commits/8c25307002c7889295d21bb72d0fe2e194450fa1 👍